### PR TITLE
impr: better handling of ghost accounts (fehmer)

### LINKED ISSUE #4909: Timeout when creating an account potentially creates a ghost account

### PR DESCRIPTION
**Pull request by fehmer**
### Description

Fixes #4909.

### CODE REVIEW

File: frontend/src/ts/popups/simple-popups.ts [hunk 1]
```diff
@@ -880,7 +880,7 @@ list.deleteAccount = new SimplePopup(
       };
     }
 
-    Notifications.add("Deleting stats...", 0);
+    Notifications.add("Deleting user...", 0);
```
Comment (Miodec) [inline frontend/src/ts/popups/simple-popups.ts:883]: I would change to something like `Deleting all data...`

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8108857745

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8123489857

**Comment by FFUV** [comment]
what does it mean by dealing with ghost accounts?

**Comment by fehmer** [comment]
build errors are unrelated and fixed with https://github.com/monkeytypegame/monkeytype/pull/5166

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**